docs/geedocs/answer/7160000.html

```diff
@@ -81,6 +81,7 @@
         </tr>
       </tbody>
     </table>
+  <p>If you're upgrading from GEE 5.1.3, you may have to reinstall the prerequisite libraries after uninstalling 5.1.3 but before installing 5.2.0.</p>
```

Comment:
This was not in the original ticket, but perhaps we should also point out that we made a number of testing updates.  That could go in the section near the top on Open Sourcing.

Reply:
To clarify, you are proposing to move the 'If you're upgrading' to a new section near 'Open Sourcing' section and in that section add 'we made a number of testing updates' ? We should probably explain 'number of testing updates'.

Reply:
I think the "If you're upgrading..." sentence can stay here.  I only put the comment here because github doesn't let you comment on code that doesn't change.

On second thought I'm going to suggest not mentioning the testing since that's something end users don't really care about.  Which means you don't need to make any changes.